public/fonts/vscode_operator_mono_lig-master/vscode_style.css

```
@@ -0,0 +1,36 @@
+.mtk7,
+.mtk3,
+.mtk4,
+.mtk13,
+.mtk16 {
+  margin-left: 1px;
+  font-family: "Operator Mono Lig";
```

Biome (1.9.4)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)